### PR TITLE
Price Plans: Display Free Trial & cleanup

### PR DESCRIPTION
**Pull request by PopDaph**
## Description

Task https://github.com/dust-tt/tasks/issues/562

Review with hidden whitespaces is highly recommended: https://github.com/dust-tt/dust/pull/4423/files?diff=split&w=1

Done: 
- [x] Edit ProPriceTable to display the button and wording when we need it.
- [x] Fix for homepage -> onclicks for Enteprise and Pro plan needs to be set for the 3 components (duplicated for responsiveness)    

Screenshot 2024-03-25 at 11 29 18

## Risk

/

## Deploy Plan

/

### CODE REVIEW

**Comment by lasryaric** [comment]
@PopDaph on the screenshot, the start now button has a display issue (look at the right padding), and I think the text should be white on a dark screen.

**Comment by PopDaph** [comment]
@lasryaric Padding fixed thank you, screenshot here: 
(Ed say blue is fine so not changing the color.)

Screenshot 2024-03-25 at 11 37 20

File: front/pages/w/[wId]/subscription/index.tsx [hunk 1]
```diff
@@ -407,41 +391,6 @@ export default function Subscription({
                 </div>
               </Page.Vertical>
             )}
-            {!plan ||
```
[the remark this answers is not in the document]
Comment (PopDaph) [reply front/pages/w/[wId]/subscription/index.tsx]: Thank you!

**Review by lasryaric — APPROVED** [review]
LGTM. Thanks

**Pull request merged**